sisyphus/delayed_ops.py

```diff
@@ -43,12 +53,21 @@ def __getitem__(self, key):
     def format(self, *args, **kwargs):
         return DelayedFormat(self, *args, **kwargs)
 
+    def rformat(self, fstring, *args, **kwargs):
+        """ Reverse format call e.g.:
+        a.rformat(b, *args, **kwargs) is mapped to b.format(a, *args, **kwargs) """
+        return DelayedFormat(fstring, self, *args, **kwargs)
+
     def replace(self, *args, **kwargs):
         return DelayedReplace(self, *args, **kwargs)
 
     def function(self, func, *args, **kwargs):
         return DelayedFunction(self, func, *args, **kwargs)
 
+    def fallback(self, fallback):
```

Comment:
Should the fallback then not be limited to "Non-Delayed" types?

Reply:
Well I guess it could be possible to stack these fallback methods, even though I don't really see a use case for it...
You are right, I either need to restrict it to Non delayed types or add try_get(...) around self.b inside DelayedFallback.get
Since I don't see much danger in allowing this usage I think we can allow it as long as it doesn't cause any problems.